install_files/ansible-base/roles/snapci/tasks/tor_config.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: add aths values to torrc
+  lineinfile:
+    backup: yes
+    dest: /etc/tor/torrc
+    line: "{{item}}"
+    regexp: "^{{item}}"
+  with_items:
+    - "{{mon_ssh_aths}}"
+    - "{{app_ssh_aths}}"
+  notify: reload tor
+  tags:
+    - tor
+
+- name: restart tor
```

Comment:
It seems unnecessary to reload **and** restart Tor after modifying the torrc. Or am I missing something?